core/src/main/scala/caliban/Rendering.scala

```diff
@@ -149,11 +149,34 @@ object Rendering {
       case _                       => ""
     }
 
-  private def renderDescription(description: Option[String], newline: Boolean = true): String = description match {
-    case None                   => ""
-    case Some(value) if newline =>
-      if (value.contains("\n")) renderTripleQuotedString("\n" + value) + "\n" else renderString(value) + "\n"
-    case Some(value)            => if (value.contains("\n")) renderTripleQuotedString(value) else renderString(value) + " "
+  private def renderDescription(description: Option[String], newline: Boolean = true): String = {
+    // Most of the graphql tools (including ours) are greedy on triple quotes. To be compatible we
```

Comment:
```suggestion
    // Most of the graphql tools are greedy on triple quotes. To be compatible we
```
Removed this because it doesn't make sense without context